packages/ui-react/src/components/Player/Player.tsx

```diff
@@ -235,11 +244,12 @@ const Player: React.FC<Props> = ({
     return () => {
       if (playerRef.current) {
         // Detaching events before component unmount
-        detachEvents();
-        playerRef.current.remove();
+        if (!backClickRef.current) {
+          playerRef.current.remove();
+        }
```

Comment:
Have you removed `detachEvents` intentionally?

We should also reset the backClickRef:
```suggestion
        if (backClickRef.current) {
          backClickRef.current = false;
          return;
        }
        playerRef.current.remove();
```

Reply:
Good idea. Thanks. I accidentally removed `detachEvents` 😅 . I restored it.